adapters/rubicon/rubicon.go

```diff
@@ -693,8 +694,14 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adap
 				continue
 			}
 
+			// if imp.ext.is_rewarded_inventory = 1, set imp.video.ext.videotype = "rewarded"
+			var videoType = ""
+			if extImpPrebid := bidderExt.Prebid; extImpPrebid != nil && extImpPrebid.IsRewardedInventory == 1 {
```

Comment:
Not that the way you did it is wrong, but is there a reason you defined `extImpPrebid` in this imp block rather than just referencing `bidderExt.Prebid` and `bidderExt.Prebid.IsRewardedInventory`?

Reply:
Actually, no reason :) Thanks for notice. Fixed.